pkgs/tools/security/munge/default.nix

```diff
@@ -21,8 +25,13 @@ stdenv.mkDerivation rec {
 
   configureFlags = [
     "--localstatedir=/var"
+    "--with-libgcrypt-prefix=${libgcrypt.dev}"
   ];
 
+  # workaround for cross compilation: https://github.com/dun/munge/issues/103
+  ac_cv_file__dev_spx = "yes";
```

Comment:
Do we want to set these unconditionally?

Reply:
DId a bit of research and found that most distributions set those to no, definitely a safer choice.

Reply:
And according to hydra logs: https://hydra.nixos.org/build/178454927/nixlog/1, these values are no at least on x86_64-linux.